tests/test_freqtradebot.py

```diff
@@ -1693,28 +1719,31 @@ def test_update_trade_state_withorderdict(default_conf, trades_for_order, limit_
     patch_exchange(mocker)
     amount = sum(x['amount'] for x in trades_for_order)
     freqtrade = get_patched_freqtradebot(mocker, default_conf)
+    caplog.clear()
     trade = Trade(
-        pair='LTC/ETH',
+        pair='LTC/USDT',
         amount=amount,
         exchange='binance',
-        open_rate=0.245441,
+        open_rate=2.0,
         open_date=arrow.utcnow().datetime,
         fee_open=fee.return_value,
         fee_close=fee.return_value,
         open_order_id="123456",
         is_open=True,
     )
-    freqtrade.update_trade_state(trade, '123456', limit_buy_order)
+    # TODO-lev: caplog.text has Amount 60.00000000000001 does not match amount 60.00000000000001
+    # TODO-lev: but they are the exact same
```

Comment:
@xmatthias  This part makes no sense, the caplog doesn't contain what it is supposed to, but if I run `pytest --pdb` and then read `caplog.text` when it breaks here, I see `Amount 60.00000000000001 does not match amount 60.00000000000001` and they are actually the exact same